stdlib/src/blackbox_fallbacks/uint32.rs

```diff
@@ -0,0 +1,620 @@
+use crate::helpers::VariableStore;
+use acir::{
+    brillig_vm::{self, RegisterIndex},
+    circuit::{
+        brillig::{Brillig, BrilligInputs, BrilligOutputs},
+        directives::QuotientDirective,
+        opcodes::{BlackBoxFuncCall, FunctionInput},
+        Opcode,
+    },
+    native_types::{Expression, Witness},
+    FieldElement,
+};
+
+/// UInt32 contains a witness that points to a field element that represents a u32 integer
+/// It has a inner field of type [Witness] that points to the field element and width = 32
+// TODO: This can be generalized to u8, u64 and others if needed.
+#[derive(Copy, Clone, Debug)]
+pub struct UInt32 {
+    pub inner: Witness,
```

Comment:
Perhaps we can create a method which is only available for tests:


```
#[cfg(test)]
pub fn get_witness(&self) -> Witness {
   self.inner
}
```

Then the inner can be private